src/OpenKNX/Common.cpp

```diff
@@ -427,33 +436,23 @@ namespace OpenKNX
         if (index >= openknx.modules.count)
             return;
 
-        openknx.modules.list[index]->loop();
+        openknx.modules.list[index]->loop(knx.configured());
     }
 
+#ifdef OPENKNX_DUALCORE
     void Common::loop1()
     {
-        if (!_setupReady)
-            return;
-
-        if (!usesDualCore())
+        if (!_setup1Ready)
```

Comment:
setup0ready should also be checked.
So loop and loop1 are both only called when both setups are ready.

Reply:
Setup1 cant be ready wirhout setup0 is Done.